strings/names/last.txt

```diff
@@ -1,851 +1,1063 @@
+	Stafford
+	Stahl
+	Stainforth
+	Stall
+	Stamos
+	Stange
+	Stavros
+	Staymates
+	Steele
+	Stephenson
+	Stern
+	Stewart
+	Stocker
 	Stoev
-	Babic
-	Nielsen
-	Jorgensen
-	Laine
+	Stone
+	Stough
+	Straub
+	Stroble
+	Stroh
+	Style
+	Su
+	Sullivan
+	Sulyard
+	Summy
+	Sun
+	Sutton
+	Suzuki
+	Swabey
+	Swarner
+	Sweetman
+	Sybilla
+	Taggart
+	Tahan
+	Takahashi
+	Tanaka
+	Tang
+	Tanner
+	Tannous
+	Tao
+	Taylor
+	Teagarden
+	Tedrow
+	Tennant
+	Thomas
+	Thomlinson
+	Thompson
+	Thomson
+	Thorley
+	Thunderhawk
+	Thursday
+	Tian
+	Tilton
+	Timm
+	Tireman
+	Titan
+	Titus
+	Todd
+	Toma
+	Tomoe
+	Torres
+	Treeby
+	Tripathi
+	Trivedi
+	Trovato
 	Tuominen
-	Poirot
-	Farkas
-	Esposito
-	Bruno
-	Caruso
-	Monti
-	Rizzi
-	Romeo
-	Jankauskas
-	Butkus
+	Turnbuckle
+	Turner
+	Tuttle
+	Ulery
+	Ullman
+	Underwood
+	Unk
+	Upadhyay
+	Vader
 	Van Dyk
 	Van Leeuwen
-	De Graaf
-	Berg
-	Olsen
-	Nowak
-	Kowalski
-	Wozniak
-	Lindberg
-	Magnusson
-	Gustafsson
-	Rossi
-	Ozdemir
-	Sahin
-	Celik
-	Lysenko
-	Melnyk
-	Fortin
-	Torres
-	Diaz
-	de Leon
+	Vance
+	Vanleer
+	Vasilyev
 	Vasquez
-	Butts
-	Nguyen
-	Kumar
-	Li
-	Zhang
-	Mirza
-	Zhao
+	Vujic
+	Wagner
+	Waldron
+	Walker
+	Wallick
+	Walmund
+	Wang
+	Ward
+	Wardle
+	Warren
+	Watanabe
+	Watson
+	Wayne
+	Webb
+	Weeter
+	Wei
+	Weinstein
+	Weisgarber
+	Wells
+	Welty
+	Wentzel
+	Werner
+	Werry
+	Wheeler
+	Whipple
+	Whirlow
+	White
+	Whiteman
+	Whittier
+	Wible
+	Wigglesworth
+	Wilcox
+	Wile
+	Wilkerson
+	Wilkinson
+	Willey
+	Williams
+	Williamson
+	Wilo
+	Wilson
+	Winton
+	Wise
+	Wolfe
+	Wolff
+	Wood
+	Woodward
+	Woodworth
+	Woolery
+	Woollard
+	Wozniak
+	Wright
+	Wu
+	Xi
+	Xiang
+	Xiao
+	Xie
 	Xu
-	Chen
-	Sun
-	Lin
-	Huang
-	Cohen
-	Katz
-	Hadad
-	Klein
-	Friedman
-	Suzuki
-	Saito
-	Nakamura
-	Ogawa
-	Kojima
-	Shibata
-	Choi
-	Yoo
-	Yee
+	Yamada
+	Yamaguchi
+	Yamamoto
+	Yamasaki
+	Yamashita
+	Yamazaki
+	Yan
 	Yang
-	Song
-	Mendis
-	Fonseca
-	Simsek
-	Ngo
-	Dang
-	Abara
-	Achebe
-	Alba
-	Dogo
-	Kenyatta
-	Obi
-	Okoye
-	Jelani
-	Jojo
-	Boro
-	Deng
-	Saleeby
-	Aziz
-	Bashar
-	Fayed
-	Osman
-	Cosmo
-	Nadir
-	Lugassi
-	Pinto
-	Gaster
-	Curie
-	Salman
+	Yee
+	Yeskey
+	Yoo
+	Yoshida
+	Young
+	Yu
+	Yuan
 	Yusuf
-	Kazem
-	Reza
-	Petrus
-	Reddy
-	Grover
-	Dhar
-	Gupta
-	Chandra
-	Shah
-	Seth
-	Modi
-	Roy
-	Bettani
+	Zadovsky
+	Zalack
 	Zand
-	Gul
-	Butt
-	Mugheri
-	Khosa
-	Dukas
-	Ioannou
-	Manos
-	Stavros
-	Montoya
-	Bird
-	Snow
-	Crowe
-	Underwood
-	Graves
-	Bishop
-	Fairbanks
-	Rhodes
-	Hammond
-	Holiday
-	Cornelius
-	Booth
-	Mendoza
-	Gold
-	Claw
-	Morton
-	Slim
-	Sparks
-	Luna
-	Page
-	Wilcox
-	Estes
-	Silver
-	Doyle
-	Guy
-	Dawes
-	Mccloud
-	Naquin
-	Carson
-	Huff
-	Griggs
-	Frost
-	Smiley
-	Barlow
-	Whipple
-	Roach
-	Lang
-	England
-	Vance
-	Beasley
-	Hobbs
-	Mathis
-	Friday
-	Thursday
-	Sellers
-	Hogan
-	Key
-	Tuttle
-	Defoe
-	Titus
-	Dee
-	Murdock
-	Cash
-	Burger
-	Hamburger
-	Salami
+	Zaun
+	Zeal
+	Zhang
+	Zhao
+	Zhong
+	Zhu
+	Zimmer
+	Zoucks
+	de Leon
+	the Eighth
+	the Fourth
+	the Third
+	the Twentieth
+	the Eleventh
+	the French
```

Comment:
Players can use these names themselves.

Reply:
Can remove these names from the pool. Only one I added here was within the joke category of, 'the French'.